examples/nlp/token_classification/conf/ner_config.yaml

```diff
@@ -7,7 +7,7 @@ trainer:
   max_steps: null # precedence over max_epochs
   accumulate_grad_batches: 1 # accumulates grads every k batches
   amp_level: O0 # O1/O2 for mixed precision
-  distributed_backend: ddp
+  distributed_backend: null
```

Comment:
ddp should stay default

Reply:
right.

Reply:
This file is gone after the merge - do you know how it's supposed to work now?

Reply:
please use token_classification.yaml, renamed it to make more general